src/plugins/console/public/application/containers/editor/monaco/monaco_editor_output_actions_provider.ts

```diff
@@ -157,13 +157,26 @@ export class MonacoEditorOutputActionsProvider {
   }
 
   public async getParsedOutput(): Promise<string> {
-    const selectedRequests = await this.getSelectedParsedOutput();
+    const model = this.editor.getModel();
+
+    if (!model) {
+      return '';
+    }
 
     let selectedRequestsString = '';
+    const selectedRequests = await this.getSelectedParsedOutput();
+
     for (const request of selectedRequests) {
-      for (const data of request.data || []) {
-        selectedRequestsString += JSON.stringify(data, null, 2) + '\n';
-      }
+      const dataString = model
```

Comment:
After we merged https://github.com/elastic/kibana/pull/189747 there were a bunch of new errors here given that most of that code changed.